tests/modules/kronatools/ktimporttaxonomy/test.yml

```diff
@@ -0,0 +1,7 @@
+- name: kronatools ktimporttaxonomy test_kronatools_ktimporttaxonomy
+  command: nextflow run tests/modules/kronatools/ktimporttaxonomy -entry test_kronatools_ktimporttaxonomy -c tests/config/nextflow.config
+  tags:
+    - kronatools/ktimporttaxonomy
+    - kronatools
+  files:
+    - path: output/kronatools/taxonomy.krona.html
```

Comment:
If the MD5 sum is not stable, it would be great to add a few keyworks with `contains` that are contained in every report.